### PR TITLE
Create decoct.md for PHP Math Functions

### LINKED ISSUE #2623: [Term Entry] PHP Math Functions decoct()
### Reason/inspiration (optional)

We would like a new term entry for [PHP](https://www.codecademy.com/resources/docs/php) about the `decoct()` function.

The entry should go in a new file in `docs/content/php/concepts/math-functions/terms/decoct/decoct.md`

The entry should 

- Have a syntax section illustrating the function with a ` ```pseudo ` code block. 
- An example showing the function in use with a ` ```php ` code block. 
- A runnable codebyte example in a ` ```codebyte/php` block. 

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by reganmarie**
### Description

- Closes #2623 
- Created decoct.md on docs/content/php/concepts/math-functions/terms/decoct/decoct.md
- Added Title, Description, Subjects, Tags, and Catalog Content
- Added description, syntax, example, and codebyte example
- Ran through 3 online plagiarism checkers to ensure 100% unique wording and description: passed with 100%
- Followed content and style guidelines -Checked for spelling and grammar errors
- Corrected for clarity and readability

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/math-functions/terms/decoct/decoct.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+Title: 'decoct()'
```
Comment (dakshdeepHERE) [inline content/php/concepts/math-functions/terms/decoct/decoct.md:2]:
```suggestion
Title: '.decoct()'
```

File: content/php/concepts/math-functions/terms/decoct/decoct.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`decoct()`** function takes a given decimal number and returns a string version of the corresponding octal value.
+
+An octal value is from the base-8 numerical system, which only uses eight digits: zero through seven.
+
+For 32-bit platforms, 4294967295 is the largest number that can be computed. For 64-bit platforms, 9223372036854775807 is the largest number that can be computed.
```
Comment (dakshdeepHERE) [inline content/php/concepts/math-functions/terms/decoct/decoct.md:19]:
```suggestion
The **`.decoct()`** function takes a given decimal number and returns a `string` version of the corresponding octal value. An octal value is from the base-8 numerical system, which only uses eight digits: zero through seven. For 32-bit platforms, 4294967295 is the largest number that can be computed. For 64-bit platforms, 9223372036854775807 is the largest number that can be computed.
```

File: content/php/concepts/math-functions/terms/decoct/decoct.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+```pseudo
+decoct($number);
+```
+
+- `number`: The decimal to convert.
+
+## Example
+
+This example uses the `decoct` function to convert a decimal to its octal value.
```
Comment (dakshdeepHERE) [inline content/php/concepts/math-functions/terms/decoct/decoct.md:31]:
```suggestion
This example uses the `.decoct()` function to convert a decimal to its octal value.
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
Few changes from my side. Otherwise looks good.

**Comment by reganmarie** [comment]
@dakshdeepHERE Made the suggested changes! 
- Updated `decoct()` to `.decoct()` in line 2, line 27, and line 43 to update per edit and for consistency
- Changed lines 15-19 to enclose string in `` and to take up one line versus being spaced out

**Review by dakshdeepHERE — APPROVED** [review]
@reganmarie this looks good for second review. 🚀

**Comment by reganmarie** [comment]
Thank you @dakshdeepHERE and @KTom101! 😊

**Review by caupolicandiaz — APPROVED** [review]
@reganmarie @dakshdeepHERE LGTM

**Pull request merged**